gis/gccview/gcc_puller_functions.py

```diff
@@ -593,59 +691,75 @@ def get_layer(mapserver_n, layer_id, schema_name, is_audited, cred = None, con =
         LOGGER.error("Invalid mapserver and/or layer Id")
         return
     #--------------------------------
-    if is_audited and primary_key is None:
-        LOGGER.error("Audited tables should have a primary key.")
-    if not(is_audited) and primary_key is not None:
-        LOGGER.error("Non-audited tables do not use the primary key.")
-    #--------------------------------
     keep_adding = True
     counter = 0
-    
-    while keep_adding == True:
+    #--------------------------------
+    if is_manual:
```

Comment:
The click command allows users to specify is-audited and primary-key, but this control structure makes manual pulls mutually exclusive with those features. Can we go back to allowing all times of pulls with use of cli?